obsah/__init__.py

```diff
@@ -555,6 +568,9 @@ def main(cliargs=None, application_config=ApplicationConfig):  # pylint: disable
 
     cli = PlaybookCLI(ansible_playbook)
     cli.parse()
+
+    rotate_log(application_config.log_path())
```

Comment:
I wonder if this should happen 3 lines above, before we do `cli = PlaybookCLI`, as that's the moment we interact with  Ansible CLI the first time.